src/System.Private.CoreLib/shared/System/Diagnostics/DebugProvider.Unix.cs

```diff
@@ -44,7 +45,7 @@ public static void FailCore(string stackTrace, string message, string detailMess
             }
         }
 
-        public static void WriteCore(string message)
+        public static void WriteCore(string? message)
```

Comment:
Why does this need to be `string?`?  Don't all callers guarantee that this is non-null, swapping in `string.Empty` if it is?